### DISCUSSION TITLE
can we PR translations for new docs ?

### DISCUSSION THREAD

**Discussion by ariaieboy** [alpinejs/alpine#1471] [3. Feature ideas]
hi i really want to translate the docs in my language i am wonder that can we PR the translations for the docs to the main website or not ?
since there is a "en" folder in the docs folder i whould really love to add "fa" version of that.
@calebporzio

**Comment by calebporzio** [comment]
Yes, please do @ariaieboy - after I merge it in, I will add support for it on the front-end of the site. Thanks!
